<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="13502" started-at="2019-12-01T06:26:07Z" finished-at="2019-12-01T06:26:21Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="13502" started-at="2019-12-01T06:26:07Z" finished-at="2019-12-01T06:26:21Z">
      <class name="MySeleniumSessionClasses.MySeleniumSessionClasses.NewTest">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:MySeleniumSessionClasses.MySeleniumSessionClasses.NewTest@40e6dfe1]" name="beforeTest" is-config="true" duration-ms="6570" started-at="2019-12-01T06:26:07Z" finished-at="2019-12-01T06:26:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="testEasy()[pri:0, instance:MySeleniumSessionClasses.MySeleniumSessionClasses.NewTest@40e6dfe1]" name="testEasy" duration-ms="6156" started-at="2019-12-01T06:26:14Z" finished-at="2019-12-01T06:26:20Z">
          <exception class="junit.framework.AssertionFailedError">
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at MySeleniumSessionClasses.MySeleniumSessionClasses.NewTest.testEasy(NewTest.java:21)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testEasy -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:MySeleniumSessionClasses.MySeleniumSessionClasses.NewTest@40e6dfe1]" name="afterTest" is-config="true" duration-ms="739" started-at="2019-12-01T06:26:20Z" finished-at="2019-12-01T06:26:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- MySeleniumSessionClasses.MySeleniumSessionClasses.NewTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
